cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4MeasureReportScorer.java

```diff
@@ -41,14 +41,18 @@ protected MeasureScoring getGroupMeasureScoring(
                     break;
                 }
             }
-            // Match by group's population id
-            if (mrgc.getPopulation().size() == entry.getKey().populations().size()) {
+            // Match by group's population id, if group-ids are not present
+            if ((mrgc.getPopulation().size() == entry.getKey().populations().size())
```

Comment:
thought: We ought to add validation for the Measure on the frontend of evaluation such that group or population is required. If both are absent we have no way to line up the scores.

Reply:
It currently errors out if missing both group & population ids, the test "noids" in measureScorerTest captures this invalid scenario. It could probably be a more meaningful error than the current "no measure scoring value set"